lib/shape.ml

```diff
@@ -58,6 +58,14 @@ let translate dx dy shape =
   | Ellipse ellipse -> Ellipse { ellipse with c = { x = ellipse.c.x + dx; y = ellipse.c.y + dy } }
   | Line line -> Line {a = {x = line.a.x + dx; y = line.a.y + dy}; b = {x = line.b.x + dx; y = line.b.y + dy}}
 
+let scale (factor:float) s =
+  let scaled_length len fact = int_of_float ((float_of_int len) *. sqrt(fact) +. 0.5) in
```

Comment:
Why is the extra `+ 0.5` needed?

Reply:
I tried rounding to the nearest integer. If it is for example 5.6. I would want it to be 6 rather 5. Is that not necessary @nikochiko ?

Reply:
Couple of changes here:
* Can you rename the function to a verb? (i.e. `scale_length`)
* Can you move that rounding logic to a separate function called `round` to make it more obvious?